contrib/qpl-cmake/doc/build.md

```diff
@@ -0,0 +1,14 @@
+This doc gives specfic build tips for QPL_Deflate codec based on Clickhouse generic [build instructions](https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/development/build.md)
+
+Several tips to build Clickhouse with QPL_deflate enabled:
```

Comment:
Hm, I guess there should be a more clear distinction between requirements of the QPL library and requirements of QPL codec support in ClickHouse. If I see correctly, the first three points affect only the QPL library. Maybe it makes sense to omit them and instead link to https://intel.github.io/qpl/documentation/get_started_docs/installation.html?